frontend/src/fonts.scss

```diff
@@ -4,17 +4,17 @@
   src: url('../node_modules/@bcgov/bc-sans/fonts/BCSans-Regular.woff') format('woff');
 }
 @font-face {
-  font-family: 'BCSans';
+  font-family: 'BCSans-Italic';
```

Comment:
I think this is always what I intended to do here, ensures consistent BCSans styling across the application for bold/italic.